fixture/bar.rb

```diff
@@ -0,0 +1,8 @@
+# typed: true
+
+class A < T::Struct
+  prop :foo, Integer
+end
+
+a = A.new(foo: 0)
+a.foo = ''
```

Comment:
What is this `fixture/` folder?

Reply:
Ah good grief `git add .` mistake I'll patch asap